2_process/src/2_process.R

```diff
@@ -0,0 +1,15 @@
+# Prepare the data for plotting
+make_eval_data <- function(data){
+  eval_data <- readr::read_csv(data, col_types = 'iccd') %>%
```

Comment:
By naming your argument `data`, I would expect what is passed in to be a data.frame. What I think is actually passed in is the filepath to the data (hence your `readr::read_csv()` call). I would suggest renaming your argument to be `data_filepath` or `data_file` or `in_file` or something along those lines to make the argument name be more descriptive for what it represents.

Reply:
agreed. wasn't paying attention at all here trying to finish the day. :D